### ISSUE TITLE
Several questions about reppoints

### ISSUE THREAD

**Issue by shwoo1219** [microsoft/RepPoints#6]
Hello,

Thanks for your great work!
I would like to use this detector in my project.

I have several questions.

1) Can the reppoints detector be adopted for class-agnostic detector (e.g. RPN)?
- I I actually quickly have tried this. However, it shows unstable training (especially high reg and cls losses for refined reppoints). Any suggestions to solve this problem? Naively reducing the learning rate does not help :(

2) Why you adjust the gradient of initial point during trainng? (i.e., multiplying 0.1 for init points)
- When I set to 1 (i.e., fully taking the gradient from subsequent stage losses), it shows degraded scores. Why does this happen?

3) For the 8 gpu training, how should I set the learning rate? 0.005? or 0.0025?

Thanks,

**Comment by yangze0930** [comment]
Hi Sanghyun, 

Thanks for your interests. 

For the first question, I have tried to use reppoints detector for detecting person in COCO dataset, and it did work. Also, I have once modified the RPN in faster RCNN to make it output a set of points and then estimate the rectangular proposals, and it also did work, I suggest to use the same sampling method as in the faster rcnn. 

For the second question, the RepPoints are driven by both recognition and object localization. This hyper-parameter balanced these two loss terms, where we empirically find 0.1 works well.

For the third question, the learning rate for 8 gpu (batch size 16) is 0.01. If you want to use a different batch size, you may need to scale the learning rate by the "Linear Scaling Rule" as suggested in [this paper](https://arxiv.org/pdf/1706.02677.pdf).

Thanks again for following up!

**Comment by shwoo1219** [comment]
Thanks for your kind reply.
